scripts/test/Calibration/tofpd/test_group_calibration.py

```diff
@@ -150,14 +171,14 @@ def test_from_prev_cal(self):
                          starting_difc[6]/(0.95*1.02),
                          starting_difc[7]*1.01], rtol=0.005)
 
-        assert_equal(mask.extractY(), [[0],
-                                       [0],
-                                       [0],
-                                       [1],
-                                       [0],
-                                       [0],
-                                       [0],
-                                       [1]])
+        assert_equal(mtd['group_calibration_pd_diffcal_mask'].extractY(), [[0],
```

Comment:
The assertion error, when it happens, will be clearer if you use `numpy.testing.assert_equal`

Reply:
I think we are just using numpy.testing.assert_equal here -- if you look at the import statement at the top of the script, we were doing from numpy.testing import assert_allclose, assert_equal there.